afl-fuzz.c

```diff
@@ -2420,6 +2422,12 @@ static u8 run_target(char** argv, u32 timeout) {
 
   if (!WIFSTOPPED(status)) child_pid = 0;
 
+  getitimer (ITIMER_REAL, &it);
```

Comment:
nit: no space in between function name and paren.
Just like the other calls in this file.

Reply:
See #23